src/coverlet.core/Reporters/CoberturaReporter.cs

```diff
@@ -32,8 +32,8 @@ public string Report(CoverageResult result)
             coverage.Add(new XAttribute("timestamp", (int)(DateTime.UtcNow - new DateTime(1970, 1, 1)).TotalSeconds));
 
             XElement sources = new XElement("sources");
-            var rootDirs = GetRootDirs(result.Modules, result.UseSourceLink).ToList();
-            rootDirs.ForEach(x => sources.Add(new XElement("source", x)));
+            var basePaths = GetBasePaths(result.Modules, result.UseSourceLink).ToList();
```

Comment:
maybe better change the name from `basePaths` to `absolutePaths`  it's more clear.

Reply:
Done